forum/communications/utils.py

```diff
@@ -0,0 +1,24 @@
+from channels.layers import get_channel_layer
+from asgiref.sync import async_to_sync
+
+
+def send_message(room_name: str, message: str):
+    channel_layer = get_channel_layer()
+
+    async_to_sync(channel_layer.group_send)(
+        room_name,
+        {"type": "chat_message", "message": message}
+    )
+
+
+def send_notification(receivers: int | list[int], message: str | dict):
```

Comment:
Ensure type hints are used correctly, especially for parameters that can take multiple types. You may use `Union` and `List` from the typing module for type hints.

Reply:
I have checked and it's the correct form of type hints since [Python 3.10
](https://peps.python.org/pep-0604/)